pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+{package} {version} {:snowflake:}
+
+**Destination channel:** {Snowflake | defaults}
+
+### Links
+
+- [{ticket_number}]() 
+- [Upstream]()
+- [conda-forge]()
+- [pypi]()
```

Comment:
Could we maybe auto-fill some of the URL or does it make sense to make it as easy as possible to copy-paste the URL?

Reply:
That's something I wouldn't mind looking into for an updated version.